src/diff.ts

```diff
@@ -0,0 +1,28 @@
+import crypto from 'crypto';
+import * as core from '@actions/core';
+import { Repo } from './github';
+import { VersionResponse } from 'bump-cli';
+import { bumpDiffComment } from './common';
+
+const commentTitle = '🤖 Documentation diff detected:';
```

Comment:
```suggestion
const commentTitle = '🤖 API change detected:';
```
I think that it's important to not talk about doc here, as we are observing structure changes.